README.md

```diff
@@ -32,7 +32,6 @@ addon | version | maintainers | summary
 [maintenance_equipment_tags](maintenance_equipment_tags/) | 15.0.1.0.0 | [![etobella](https://github.com/etobella.png?size=30px)](https://github.com/etobella) | Adds category tags to equipment
 [maintenance_plan](maintenance_plan/) | 15.0.1.2.0 |  | Extends preventive maintenance planning
 [maintenance_plan_activity](maintenance_plan_activity/) | 15.0.1.0.2 |  | This module allows defining in the maintenance plan activities that will be created once the maintenance requests are created as a consequence of the plan itself.
-[maintenance_product](maintenance_product/) | 15.0.1.0.0 | [![victoralmau](https://github.com/victoralmau.png?size=30px)](https://github.com/victoralmau) | Maintenance Product
```

Comment:
Why are you editing this?

Reply:
It gives a conflict, probably I did something wrong. I will try to repair it.